include/rdma/fabric.h

```diff
@@ -481,6 +481,10 @@ struct fi_param {
 int fi_getparams(struct fi_param **params, int *count);
 void fi_freeparams(struct fi_param *params);
 
+#if defined(_WIN32)
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
+#endif
```

Comment:
Please move this to the top of the file, so that future changes don't use ssize_t before it's defined on windows.